test/semantics/resolve35.f90

```diff
@@ -88,7 +88,6 @@ subroutine s7
   do concurrent(integer::i=1:5) local(j, i) &
       !ERROR: 'j' is already declared in this scoping unit
       local_init(k, j) &
-      !ERROR: Variable 'a' not found
       shared(a)
     a(i) = j + 1
```

Comment:
How will `a` be dimensioned?

Reply:
Currently, the code creates a variable in the enclosing scope that's a scalar and creates another variable whose scope is the block of the ```DO CONCURRENT``` that's "host associated" with it.

Reply:
But the test applies subscripts to `a`.